### DISCUSSION TITLE
Do you accept icon submissions?

### DISCUSSION THREAD

**Discussion by kevincannon** [phosphor-icons/homepage#619] [Q&A] [answered]
We're going to switch our app to Phosphor, but there are some icons missing, so we'll be redrawing them to fit the Phosphor style. Do you accept community submissions for icons?

If so, what's the process?

Thanks.

**Comment by rektdeckard** [comment, marked as answer]
Hey @kevincannon! Typically no, we don't accept icon submissions. We're very particular about each and every icon, and there are a number of design intuitions we follow that are difficult to express in writing. That said, a number of our libraries have some conveniences around [custom icons](https://github.com/phosphor-icons/react?tab=readme-ov-file#custom-icons).

**Reply by kevincannon** [reply]
Thanks so much for the reply. I can totally understand that, it's such a well-crafted icon set! 👏

Cheers for pointing me to the Custom Icons option, I hadn't seen that. That will help us wrap those custom ones up nicely.

All the best with everything, thanks again for putting together and sharing such a nice set of icons!

**Reply by rektdeckard** [reply]
Our pleasure!

**Comment by pierrebaronqumu** [comment]
@rektdeckard, Do you have more information on how to design the icon in the design tool? 

Things like:
* line weight for each style.
* spacing
* roundness

Thank you!

**Reply by pierrebaronqumu** [reply]
I guess I am asking for something similar to https://lucide.dev/contribute/icon-design-guide.
